app/index-dev.html

```diff
@@ -22,12 +23,26 @@
 
         <p>
           <span>Please consult the repository's README for more information:</span><br>
-          <a href='https://github.com/brave/browser-electron/blob/master/README.md'>https://github.com/brave/browser-electron/blob/master/README.md</a>
+          <a href="https://github.com/brave/browser-electron/blob/master/README.md">https://github.com/brave/browser-electron/blob/master/README.md</a>
         </p>
       </div>
     </div>
   </body>
-  <script src="http://localhost:8080/webpack-dev-server.js"
-          onerror="document.querySelector('#setupError').style.display ='block'"></script>
-  <script src="http://localhost:8080/built/bundle.js"></script>
+  <script>
+    const createScript = function (scriptPath) {
+      return new Promise(function (resolve, reject) {
+        var script = document.createElement("script")
+        script.type = "text/javascript"
```

Comment:
Yeah sure, I agree with this actually - generally I try to adopt the conventions in the file - the issue was that it didn't seem consistent already (requires for example used double above) so I just made a call.  I'll adjust.